README.md

```diff
@@ -371,182 +286,125 @@ Unzip the file. It should be renamed to just usagov.sql. Place that uncompressed
-
+This project's default start procedure (`docker compose up`) will start a container to automatically watch for changes and recompile the theme as needed.
 
 The theme can be manually built at any time through gulp's build task. Any other gulp task can be triggered the same way.
-
-
 ```
 # Rebuild theme
 bin/npm run build
 ```
 
-
 Any changes made to the node modules needed for building the theme will require a re-install of the node_modules before build.
-
-
 ```
 # Reinstall node modules
 bin/npm install
 bin/npm run build
 ```
 
-
-
-
 This theme adds `USWDS_CKEditor_Custom_Styles.scss` into the CKeditor frame.
 
-
-
-
 [back to top](#usagov-2021)
 
-
 ## Export Database
 A helper script has been provided to perform exports.
-
-
 ```
 bin/db-export
 ```
 
-
 You may specify a filename if you don't want to overwrite the default file location with a new export.
-
-
 ```
 bin/db-export other-backup.sql
 ```
 
-
 This process asks drush to export the database for us since it does some cleanup work before running the export.
 
-
-
-
 [back to top](#usagov-2021)
 
-
 ## Export Config
 
-
 1. View differences
-   * Configuration > Development > Configuration Synchronization
-   * `/admin/config/development/configuration`
+   * [Configuration > Development > Configuration Synchronization](http://localhost/admin/config/development/configuration)
 2. Export
    * via Command Line
-      a. `bin/drush cex`
+      - `bin/drush cex`
    * via Export Full Archive
-      a. Export > Full Archive
-      b. Move the desired configs into `/config/sync`
+      - [Export > Full Archive](http://localhost/admin/config/development/configuration/full/export)
+      - Move the desired configs into `/config/sync`
    * via Export Single Item
-      a. Export > Single Item
-      b. Find the config you want to sync
-      c. Create/Edit the file in `/config/sync` with the filename shown below the config textbox
-      d. Paste the config text into the file
-      e. Repeat for each desired config
+      - [Export > Single Item](http://localhost/admin/config/development/configuration/single/export)
+      - Find the config you want to sync
+      - Create/Edit the file in `/config/sync` with the filename shown below the config textbox
+      - Paste the config text into the file
+      - Repeat for each desired config
 3. Commit config changes to git
 
-
-
-
 [back to top](#usagov-2021)
 
-
 ## Import Config
 
-
 `bin/drush cim`
 
-
-
-
 [back to top](#usagov-2021)
 
-
 ## Build and Deploy procedure
-Production ready containers can be built and deployed from a local environment. To do so, proper secrets must be entered into the env.local file as environmental variables. This same procedure is used by CircleCI and is defined in .circleci/config.yml
-
-
-
+Production ready containers can be built and deployed from a local environment.
```

Comment:
Please remove "Production ready" here, and add "to the dev environment or to the developer's sandbox space."